lib/src/verifiers/unicity/endpoints.spec.ts

```diff
@@ -64,7 +64,7 @@ describe("unique endpoint names verifier", () => {
     };
     expect(verifyUniqueEndpointNames(contractNode)).toMatchObject([
       {
-        message: "endpoints must have unique names"
+        message: "endpoints must have unique names: EndpointOne"
```

Comment:
I think a better wording now would be `Found multiple endpoints with the name "EndpointOne"`. Thoughts? (same for types)

Reply:
Would be worth considering for a complete rethink of error messages, but currently error messages state the rule that the contract must follow. For now I think consistency is best

Reply:
SGTM!